files/flowthings/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "npm",
+  "name": "flowthings-browser",
+  "repo": "flowthings/browser-client",
+  "files": {
+    "basePath": "dist/",
+    "include": ["flowthings-browser.js", "flowthings-browser.min.js"]
```

Comment:
I dont think that `flowthings-browser.js` is needed in production. Can you remove it?